code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -26,7 +26,7 @@
 
 /datum/species/zombie/infectious
 	name = "Infectious Zombie"
-	id = "memezombies"
+	id = "romerolzombies"
```

Comment:
no i need to fool new trial admins to give me romerol zombie this is a bad change